Formula/jabba.rb

```diff
@@ -1,43 +1,27 @@
 class Jabba < Formula
   desc "Cross-platform Java Version Manager"
-  homepage "https://github.com/shyiko/jabba"
-  url "https://github.com/shyiko/jabba/archive/0.11.2.tar.gz"
-  sha256 "33874c81387f03fe1a27c64cb6fb585a458c1a2c1548b4b86694da5f81164355"
+  homepage "https://github.com/Jabba-Team/jabba"
+  url "https://github.com/Jabba-Team/jabba/archive/0.12.0.tar.gz"
+  sha256 "15a142239869733d7f0fe8c0cc0cd99f619e5bc8121ebabc9c28c382333b89c0"
```

Comment:
This change doesn't seem to be advertised anywhere upstream. Where can we find out about this?

Reply:
Original upstream (https://github.com/shyiko/jabba) has no activity for more than a year, especially regarding the index of Java distribution; as jabba is a Java distribution manager, it needs to be kept up-to-date.

A fork has been created to provide both a living codebase, starting from the old one, and an up-to-date distributions index (https://github.com/Jabba-Team/jabba)

Reply:
For security reasons, unless the original author or other package managers deemed this project the official successor somewhere public, this should be submitted as a new formula.

Reply:
Understood.
Where can I find a recommendation for an alternate name for a new formula in this case?

Reply:
A few of discuss forking and one of the original maintainers  (willcohen) confirms Jabba-Team/ as the new home of the project here: https://github.com/shyiko/jabba/issues/833#issuecomment-1338648294

Would that be enough to confirm the change of repo?

Reply:
Yeah, that looks fine. Can you reference that in the formula file @nicerloop ?

Reply:
Added a link to the discussion comment where the former maintainer blessed the fork